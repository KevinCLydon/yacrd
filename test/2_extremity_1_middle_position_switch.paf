1	1000	450	100	-	2	1000	550	900	350	350	255
1	1000	550	900	-	3	1000	450	100	350	350	255
