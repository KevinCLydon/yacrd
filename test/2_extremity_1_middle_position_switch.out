Chimeric	1	1000	100,0,100;100,450,550;100,900,1000
